Resources/Prototypes/Limbs/human.yml

```diff
@@ -0,0 +1,128 @@
+- type: limb
+  id: HumanHead
+  name: Human Head
+  health: 100
+  dollIcon: Human/head.png
+  prototype: HumanHead
+  target: enum.AttackTargetDef.Head
```

Comment:
I believe you removed all this attack targeting code so you should remove it from the prototypes too.